Ball.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Ball {
+    private Color color;
+    private int number;
+    public Ball (Color color, int number) {

```

Comment:
add empty line before. There are many formatting mistakes in the files - please use auto formatting